x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/translations.ts

```diff
@@ -157,7 +157,7 @@ export const referenceErrorMessage = (referenceCount: number) =>
 export const EXCEPTION_LIST_SEARCH_PLACEHOLDER = i18n.translate(
   'xpack.securitySolution.exceptions.searchPlaceholder',
   {
-    defaultMessage: 'e.g. Example List Name',
```

Comment:
As this message text changes, should we also remove the old text from the translation files?

Reply:
I updated the key, so yea I think I'll need to remove the old one from the translation files.